### ISSUE TITLE
Update to last version and websocket wss://ws-api.exmo.com:443/v1/public do not work

### ISSUE THREAD

**Issue by kleninmaxim** [Textalk/websocket-php#167]
**Describe the bug**
I write as an example a simpe code:
```php
<?php

use WebSocket\Client;

require __DIR__ . '/vendor/autoload.php';

$client = new Client("wss://ws-api.exmo.com:443/v1/public");
$client->text("Hello WebSocket.org!");
print_r($client->receive());
echo PHP_EOL;
print_r('done'); echo PHP_EOL; die();
```
When I started it, it fails and get error:
`PHP Fatal error:  Uncaught ErrorException: stream_socket_client(): Unable to connect to ssl://ws-api.exmo.com (Failed to parse address "ws-api.exmo.com") in ../vendor/textalk/websocket/lib/Client.php:348`

Class error:
```php
WebSocket\ConnectionException Object
(
    [message:protected] => Could not open socket to "ws-api.exmo.com": stream_socket_client(): Unable to connect to ssl://ws-api.exmo.com (Failed to parse address "ws-api.exmo.com") (0).
    [string:Exception:private] => 
    [code:protected] => 0
    [file:protected] => ../vendor/textalk/websocket/lib/Client.php
    [line:protected] => 362
    [trace:Exception:private] => Array
        (
            [0] => Array
                (
                    [file] => ../vendor/textalk/websocket/lib/Client.php
                    [line] => 169
                    [function] => connect
                    [class] => WebSocket\Client
                    [type] => ->
                )

            [1] => Array
                (
                    [file] => ../vendor/textalk/websocket/lib/Client.php
                    [line] => 130
                    [function] => send
                    [class] => WebSocket\Client
                    [type] => ->
                )


        )
...
```

In previous versions (May be 1.5) it work correct.
Websocket `wss://ws-api.exmo.com:443/v1/public` is correct [(here example)](https://github.com/exmo-dev/web-socket-api-php-client/tree/master)

**Versions**
* Version of this library - 1.6.1
* PHP version - 8.1

**Comment by sirn-se** [comment]
I think I see the problem. Will patch later on today.

**Comment by sirn-se** [comment]
Fixed in `1.6.2`. It was the specification of PSR UriInterface that caused the port number to be excluded. Now fixed.

**Comment by kleninmaxim** [comment]
> Fixed in `1.6.2`. It was the specification of PSR UriInterface that caused the port number to be excluded. Now fixed.

Thanks!
